diff -Nur -x.git -xdocs '-x*mcbane87.config' -xextra_includes -xstandalone.patch -x.gitignore old/android/Kbuild new/android/Kbuild
--- old/android/Kbuild	1970-01-01 01:00:00.000000000 +0100
+++ new/android/Kbuild	2015-10-09 20:48:27.748171671 +0200
@@ -0,0 +1,11 @@
+# DO NOT EDIT. This file is generated from Kbuild.src
+# Makefile for busybox
+#
+# Copyright (C) 1999-2005 by Erik Andersen <andersen@codepoet.org>
+#
+# Licensed under GPLv2, see file LICENSE in this source tree.
+
+lib-y:=
+
+lib-y  += android.o
+lib-y  += reboot.o
diff -Nur -x.git -xdocs '-x*mcbane87.config' -xextra_includes -xstandalone.patch -x.gitignore old/android/libc/Kbuild new/android/libc/Kbuild
--- old/android/libc/Kbuild	1970-01-01 01:00:00.000000000 +0100
+++ new/android/libc/Kbuild	2015-10-09 20:48:27.748171671 +0200
@@ -0,0 +1,12 @@
+# DO NOT EDIT. This file is generated from Kbuild.src
+# Makefile for busybox
+#
+# Copyright (C) 1999-2005 by Erik Andersen <andersen@codepoet.org>
+#
+# Licensed under GPLv2, see file LICENSE in this source tree.
+
+lib-y:=
+
+lib-y  += mktemp.o
+lib-y  += pty.o
+lib-y  += __set_errno.o
diff -Nur -x.git -xdocs '-x*mcbane87.config' -xextra_includes -xstandalone.patch -x.gitignore old/android/librpc/Kbuild new/android/librpc/Kbuild
--- old/android/librpc/Kbuild	1970-01-01 01:00:00.000000000 +0100
+++ new/android/librpc/Kbuild	2015-10-09 20:48:27.752171671 +0200
@@ -0,0 +1,52 @@
+# DO NOT EDIT. This file is generated from Kbuild.src
+# Makefile for busybox
+#
+# Copyright (C) 1999-2005 by Erik Andersen <andersen@codepoet.org>
+#
+# Licensed under GPLv2, see file LICENSE in this source tree.
+
+lib-y:=
+
+lib-y += pm_getmaps.o
+lib-y += rpc_thread.o
+lib-y += clnt_udp.o
+lib-y += clnt_perror.o
+lib-y += clnt_simple.o
+lib-y += auth_unix.o
+lib-y += xdr_stdio.o
+lib-y += pmap_prot.o
+lib-y += xdr_rec.o
+lib-y += svc_authux.o
+lib-y += pmap_clnt.o
+lib-y += xdr_reference.o
+lib-y += rpc_prot.o
+lib-y += rpc_dtablesize.o
+lib-y += bindresvport.o
+lib-y += rpc_cmsg.o
+lib-y += svc.o
+lib-y += pm_getport.o
+lib-y += svc_udp.o
+lib-y += rpc_commondata.o
+lib-y += xdr.o
+lib-y += svc_run.o
+lib-y += auth_none.o
+lib-y += svc_tcp.o
+lib-y += xdr_float.o
+lib-y += clnt_raw.o
+lib-y += authunix_prot.o
+lib-y += pmap_prot2.o
+lib-y += clnt_tcp.o
+lib-y += xdr_intXX_t.o
+lib-y += pmap_rmt.o
+lib-y += clnt_unix.o
+lib-y += xdr_mem.o
+lib-y += getrpcent.o
+lib-y += get_myaddress.o
+lib-y += svc_simple.o
+lib-y += svc_unix.o
+lib-y += getrpcport.o
+lib-y += create_xid.o
+lib-y += svc_auth.o
+lib-y += clnt_generic.o
+lib-y += xdr_array.o
+lib-y += svc_raw.o
diff -Nur -x.git -xdocs '-x*mcbane87.config' -xextra_includes -xstandalone.patch -x.gitignore old/android/regex/Kbuild new/android/regex/Kbuild
--- old/android/regex/Kbuild	1970-01-01 01:00:00.000000000 +0100
+++ new/android/regex/Kbuild	2015-10-09 20:48:27.764171670 +0200
@@ -0,0 +1,10 @@
+# DO NOT EDIT. This file is generated from Kbuild.src
+# Makefile for busybox
+#
+# Copyright (C) 1999-2005 by Erik Andersen <andersen@codepoet.org>
+#
+# Licensed under GPLv2, see file LICENSE in this source tree.
+
+lib-y:=
+
+lib-y += bb_regex.o
diff -Nur -x.git -xdocs '-x*mcbane87.config' -xextra_includes -xstandalone.patch -x.gitignore old/android/selinux/Kbuild new/android/selinux/Kbuild
--- old/android/selinux/Kbuild	1970-01-01 01:00:00.000000000 +0100
+++ new/android/selinux/Kbuild	2015-10-09 20:48:27.768171670 +0200
@@ -0,0 +1,11 @@
+# DO NOT EDIT. This file is generated from Kbuild.src
+# Makefile for busybox
+#
+# Copyright (C) 1999-2005 by Erik Andersen <andersen@codepoet.org>
+#
+# Licensed under GPLv2, see file LICENSE in this source tree.
+
+lib-y:=
+
+lib-y += matchpathcon.o
+lib-y += stubs.o
diff -Nur -x.git -xdocs '-x*mcbane87.config' -xextra_includes -xstandalone.patch -x.gitignore old/include/libbb.h new/include/libbb.h
--- old/include/libbb.h	2015-10-14 21:54:42.311690254 +0200
+++ new/include/libbb.h	2015-10-09 20:48:27.896171667 +0200
@@ -1985,4 +1985,8 @@
 
 POP_SAVED_FUNCTION_VISIBILITY
 
+#ifndef CONFIG_DEFAULT_MODULES_DIR
+#define CONFIG_DEFAULT_MODULES_DIR "/system/lib/modules"
+#endif
+
 #endif
diff -Nur -x.git -xdocs '-x*mcbane87.config' -xextra_includes -xstandalone.patch -x.gitignore old/include/platform.h new/include/platform.h
--- old/include/platform.h	2015-10-14 21:54:42.311690254 +0200
+++ new/include/platform.h	2015-10-09 20:48:27.900171667 +0200
@@ -381,7 +381,7 @@
 #define HAVE_VASPRINTF 1
 #define HAVE_UNLOCKED_STDIO 1
 #define HAVE_UNLOCKED_LINE_OPS 1
-#define HAVE_GETLINE 1
+//#define HAVE_GETLINE 1
 #define HAVE_XTABS 1
 #define HAVE_MNTENT_H 1
 #define HAVE_NET_ETHERNET_H 1
diff -Nur -x.git -xdocs '-x*mcbane87.config' -xextra_includes -xstandalone.patch -x.gitignore old/Makefile new/Makefile
--- old/Makefile	2015-10-14 21:54:42.159690257 +0200
+++ new/Makefile	2015-10-09 20:48:27.728171671 +0200
@@ -4,12 +4,6 @@
 EXTRAVERSION =
 NAME = bionic
 
-# prevent local tree builds in bionic,
-# but allow initial version check (SUBMAKE)
-ifeq (,$(filter s, $(MAKEFLAGS)))
-    KBUILD_OUTPUT ?= $(OUT)/obj/busybox
-endif
-
 # *DOCUMENTATION*
 # To see a list of typical targets execute "make help"
 # More info can be located in ./README
@@ -471,6 +465,9 @@
 		applets/ \
 
 libs-y		:= \
+		android/ \
+		android/libc/ \
+		android/regex/ \
 		archival/ \
 		archival/libarchive/ \
 		console-tools/ \
diff -Nur -x.git -xdocs '-x*mcbane87.config' -xextra_includes -xstandalone.patch -x.gitignore old/networking/nslookup.c new/networking/nslookup.c
--- old/networking/nslookup.c	2015-10-14 21:54:42.367690252 +0200
+++ new/networking/nslookup.c	2015-10-14 20:37:45.268433677 +0200
@@ -25,7 +25,11 @@
 //usage:       "Name:       debian\n"
 //usage:       "Address:    127.0.0.1\n"
 
+#ifdef KK
+#include <resolv_kk.h> 
+#else
 #include <resolv.h>
+#endif
 #include "libbb.h"
 
 #ifdef ANDROID
Binärdateien old/_prebuilds/busybox_arm und new/_prebuilds/busybox_arm sind verschieden.
